examples/gallery.rst

```diff
@@ -48,6 +48,16 @@ Gaussian Processes
 
     gaussian_processes/*
 
+BART
+----
+
+.. nbgallery::
+    :name: BART
+    :glob:
+    :reversed:
+
+    BART/*
```

Comment:
Isn't BART moved to experimental anyway?

Reply:
yes, but examples with experimental should also be here in principle